app/views/clients/rollup/_custom_services.html.haml

```diff
@@ -0,0 +1,13 @@
+:ruby
+  custom_services = @client.custom_services.
+    joins(enrollment: :client).
+    joins(custom_service_type: :custom_service_category).
+    merge(GrdaWarehouse::Hud::Enrollment.visible_to(current_user)).
+    preload( :enrollment, :project, :client, :custom_service_type ).
+    preload(custom_service_type: :custom_service_category).
```

Comment:
I don't think it affects the sql at all, but you can do this in one line:
```
preload(enrollment: [:project, :client], custom_service_type: [:custom_service_category])
```

I don't know if we really need to preload the client though..it seems silly since we are already on the client dashboard. It looks like it's used for the data source display somehow? I will leave a comment down there :)